urbanterror.appdata.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
+<component type="desktop">
+  <id>urbanterror.desktop</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-2.0+ AND LicenseRef-proprietary=https://www.urbanterror.info/support/202-quake-3-sdk/</project_license>
```

Comment:
it is GPL or is it proprietary?

Note: game data isn't considered here.

Reply:
Engine is GPL, game scripts and assets are under this weird Quake III SDK license which only allows digital distribution for free.